<?xml version="1.0"?>
<launch>

  <node pkg="localization" type="localization_node" respawn="false" name="localization" output="screen">
    <rosparam file="$(find localization)/config/localization_config.yaml" command="load" />
  </node>

  <!-- <node name="rviz_localization" pkg="rviz" type="rviz" args="-d $(find localizasstion)/src/ros_viewer/localization.rviz"/> -->
  
</launch>